src/sass/components/_post-box.scss

```diff
@@ -33,7 +52,7 @@
     padding: 25px;
 
     h5 {
-      color: $primary;
+      color: rgb(103, 103, 103);
```

Comment:
ten kolor się chyba powtarza w appce - może wrzucić to do jakiejś sensownej zmiennej?